src/index.html

```diff
@@ -144,7 +144,7 @@ <h3>Directions</h3>
       </article>
 
       <article class = "pantry hidden">
-          <h2 class="recipe-title">User's Pantry</h2>
+          <h2 class="recipe-title"><span id="pantry-user-name">User</span>'s Pantry</h2>
```

Comment:
I like using span for User's name, this way we refactor and replace them all together later